src/Components/Playground/Playground.tsx

```diff
@@ -20,15 +21,42 @@ const emptyPlayground = [
   [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
   [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
   [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-  [0, 0, 0, 1, 0, 1, 0, 0, 0, 0],
-  [0, 0, 1, 0, 0, 0, 0, 0, 0, 0],
-  [1, 1, 1, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
 ];
 export const Playground = () => {
-  const [playground] = useState(emptyPlayground);
-  const [figure] = useState(getRandomFigure());
+  const handleKeyPress = (e: KeyboardEvent) => {
+    const buttonKey = e.code;
+    console.log(e.code);
 
-  console.log(figure.coords);
+    let movedFigure: Figure;
+    switch (buttonKey) {
+      case 'ArrowDown':
+        movedFigure = moveFigureDown(figure);
+        setFigure(movedFigure);
```

Comment:
`movedFigure` variable is redundant. Removing it reduces the code by about 5 lines.
```suggestion
        setFigure(moveFigureDown(figure));
```